csharp/ql/src/semmle/code/csharp/dataflow/internal/rangeanalysis/SignAnalysisSpecific.qll

```diff
@@ -75,6 +75,11 @@ predicate unknownIntegerAccess(Expr e) {
   // property access
   e instanceof CS::PropertyAccess and e.getType() instanceof NumericOrCharType
   or
+  // field access on non-this instance
```

Comment:
@hvitved We have data flow analysis on fields. It seems to me that it's limited to access on the `this` instance. Are there any other limitations? Do we have analysis on static fields?

Reply:
I am not sure why we need this case; `fieldSign` should handle all cases regardless of qualifiers or whether the field happens to be `static`.